src/server/twemcache-rs/src/server.rs

```diff
@@ -0,0 +1,283 @@
+// Copyright 2021 Twitter, Inc.
```

Comment:
With the understanding that we will (soon) consolidate common event/IO logic between different servers, I'm skipping this file.